publications-2018.bib

```diff
@@ -886,4 +886,30 @@ @article{kaufl2018
   doi = {10.1093/gji/ggy375}
 }
 
+@Article{Ludvigsson2018,
+  author={Ludvigsson, Gustav and Steffen, Kyle R. and Sticko, Simon and Wang, Siyang and Xia, Qing and Epshteyn, Yekaterina and Kreiss, Gunilla},
+  title={High-Order Numerical Methods for 2D Parabolic Problems in Single and Composite Domains},
+  journal={Journal of Scientific Computing},
+  year={2018},
+  month={Aug},
+  volume={76},
+  number={2},
+  pages={812--847},
+  issn={1573-7691},
+  doi={10.1007/s10915-017-0637-y},
+  url={https://doi.org/10.1007/s10915-017-0637-y}
+}
+
+﻿@phdthesis{StickoThesis2018,
```

Comment:
this line contains a non-ASCII character, can you double-check (I think in front of the @)

Reply:
Oops, strange copy-paste mistake. Fixed it. :)